tests/test_text/test_dispersion.py

```diff
@@ -0,0 +1,76 @@
+# tests.test_text.test_dispersion
+# Tests for the dispersion plot visualization
+#
+# Author:   Larry Gray
+# Github:   @lwgray
+# Created:  2018-06-22 15:27
+#
+# Copyright (C) 2018
+# For license information, see LICENSE.txt
+#
+# ID: test_dispersion.py [] lwgray@gmail.com $
+
+"""
+Tests for the dispersion plot text visualization
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+
+from yellowbrick.text.dispersion import *
+from tests.dataset import DatasetMixin
+from tests.base import VisualTestCase
+from itertools import chain
+
+##########################################################################
+## DispersionPlot Tests
+##########################################################################
+
+class DispersionPlotTests(VisualTestCase, DatasetMixin):
```

Comment:
It looks like AppVeyor is having a problem with the hobbies corpus, which I'll add an issue about. In the meantime, could you add the following:

```python
@pytest.mark.xfail(sys.platform == "win32", reason="Issue #491")
class DispersionPlotTests(VisualTestCase, DatasetMixin):
    ...
```